sockeye/arguments.py

```diff
@@ -587,7 +587,11 @@ def add_inference_args(params):
 
     decode_params.add_argument('--output-type',
                                default='translation',
-                               choices=["translation", "translation_with_alignments", "align_plot", "align_text"],
+                               choices=["translation",
```

Comment:
with the growing number of options here it looks like its time to adopt the pattern of defining a C.OUTPUT_HANDLER_CHOICES defined in constants.py as well as setting default to a defined constant.